sparse/_coo/core.py

```diff
@@ -1561,6 +1561,53 @@ def swapaxes(self, axis1, axis2):
         axes[axis1], axes[axis2] = axes[axis2], axes[axis1]
         return self.transpose(axes)
 
```

Comment:
I'd add a comment here indicating where the code was taken from, including a link and line numbers.